test/customerBookingsTest.js

```diff
@@ -0,0 +1,86 @@
+import chai from 'chai';
+const expect = chai.expect;
+
+import { sampleData } from '../test/sampleData.js';
+import { getCustomerBookings, calculateTotalCost } from '../src/customerBookings.js'
+
+describe('getCustomerBookings', () => {
+  it('should return all bookings for a customer with multiple bookings', () => {
+    const customerId = 1;
+    const expectedBookings = [
+      { "id": "5fwrgu4i7k55hl6sz", "userID": 1, "date": "2022/04/22", "roomNumber": 1 },
+      { "id": "5fwrgu4i7k55hl6t8", "userID": 1, "date": "2023/01/25", "roomNumber": 1 }
+    ];
+    const customerBookings = getCustomerBookings(customerId, sampleData.bookings);
+    expect(customerBookings).to.deep.equal(expectedBookings);
+  });
+
+  it('should return a single booking for a customer with one booking', () => {
+    const customerId = 5;
+    const expectedBookings = [
+      {"id": "5fwrgu4i7k55hl6t7", "userID": 5, "date": "2023/01/19", "roomNumber": 5}
+    ];
+    const customerBookings = getCustomerBookings(customerId, sampleData.bookings);
+    expect(customerBookings).to.deep.equal(expectedBookings);
+  });
+
+  it('should return an empty array for a customer with no bookings', () => {
+    const customerId = 55;
+    const customerBookings = getCustomerBookings(customerId, sampleData.bookings);
+    expect(customerBookings).to.be.an('array').that.is.empty;
+  });
+
+  it('should handle invalid customer ID types', () => {
+    const invalidCustomerIds = [null, undefined, 'string', {}, []];
+    invalidCustomerIds.forEach(invalidId => {
+      const customerBookings = getCustomerBookings(invalidId, sampleData.bookings);
+      expect(customerBookings).to.be.an('array').that.is.empty;
+    });
+  });
+
+  it('should return an empty array if the bookings data is empty', () => {
+    const customerId = 1;
+    const customerBookings = getCustomerBookings(customerId, []);
+    expect(customerBookings).to.be.an('array').that.is.empty;
+  });
+
+});
+
+describe('calculateTotalCost', () => {
+  
+  it('should calculate the total cost for a customer with multiple bookings', () => {
+    const customerId = 1;
+    const customerBookings = getCustomerBookings(customerId, sampleData.bookings);
+    const totalCost = calculateTotalCost(customerBookings, sampleData.rooms);
+    const expectedCost = 358.4 * 2;
```

Comment:
I would set the expectedCost equal to the actual expected number since the output will be a single number (not an equation), kind of nitpicky but perhaps a better practice.

Reply:
I agree.